LeMinhQuoc/blog-frontend
# Feature/login form

- Solve : login Form
- How dose it PR solve it
    +add login form
    +call login_check after login
    +render home page
- How can reviewer test the changes
    +Pull this branch on your local.
    +Install API source
    +Run composer install.
    +Run symfony server:start
    +Flow https://localhost/ and click a blog's title to check
